src/vaults/vaults.service.ts

```diff
@@ -147,9 +147,8 @@ export class VaultsService {
     vault.lastHarvest = pendingHarvest.lastHarvestedAt;
 
     const harvestProjection = this.getVaultYieldProjection(vault, pendingHarvest);
-    const passiveSources = sourcesApy.filter(
-      (s) => s.type.includes(SourceType.TradeFee) || s.type.includes(SourceType.Emission),
-    );
+    // filter out all vault compound sources - this include 'precompound' as well
```

Comment:
if it includes preCompound as well what is the sourceType.preCompound for?

Reply:
or is the comment meaning we want everything but compound because if so the comment doesn't help much